content/lessons/genesis/genesis-5/index.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { useTranslations } from 'hooks'
+import { CodeExample, LessonInfo, Title, Text, TerminalChallenge } from 'ui'
+
+export default function Genesis3({ lang }) {
+  const t = useTranslations(lang)
+
+  return (
+    <TerminalChallenge
+      expectedInput={{
+        userVariable: '$scriptSigHex',
+        value:
```

Comment:
Not sure if this is just github being weird, but formatting seems off here.

Reply:
I have formatting on save and it seems to be running correctly, I think due to the length of the string it is formatting it to the next line